File: 263. Ugly Number/isUgly.py
class Solution:
  def isUgly(self, n: int) -> bool:
    if n <= 0:
      return False

    primeNumbers = [2, 3, 5]
    for pn in primeNumbers:
      while n % pn == 0:
        n = n / pn

    return n == 1

File: 263. Ugly Number/isUgly.js
/**
 * @param {number} n
 * @return {boolean}
 */
const isUgly = (n) => {
  if (n <= 0) {
    return false;
  }

  primeNumbers = [2, 3, 5];
  for (let i = 0; i < primeNumbers.length; i++) {
    while (n % primeNumbers[i] === 0) {
      n = n / primeNumbers[i];
    }
  }

  return n === 1
};

console.log('isUgly: ', isUgly(6));